docs/minio-client-complete-guide.md

```diff
@@ -1479,11 +1479,25 @@ ENVIRONMENT VARIABLES:
 
 ```
 mc stat play/mybucket
-Name      : mybucket/
-Date      : 2018-02-06 18:06:51 PST
-Size      : 0B
-Type      : folder
-```
+Encryption: Not Set
+
```

Comment:
may need a better formatting @poornas - I think the previous approach is fine instead these features could be `Metadata` field..

```
~ mc stat play/testbucket
Name      : testbucket/
Date      : 2020-08-06 21:09:46 PDT 
Size      : 0 B
Type      : folder 
Metadata  :
  Versioning: Enabled
  LockConfiguration: Configured
  Replication: Disabled
  Notification: Configured
  Policy: upload
  Location: us-west-1
  Tagging: key1=value1, key2=value2
  ILM: Configured
```

Reply:
^^ is this discussed? @poornas @kannappanr - IMO we should avoid different format